pkg/relayer/config/types.go

```diff
@@ -173,6 +174,11 @@ type RelayMinerSupplierServiceConfig struct {
 	// authentication then this field must be populated accordingly.
 	// For example: { "Authorization": "Bearer <token>" }
 	Headers map[string]string
+	// ForwardIdentityHeaders determines if identity headers should be forwarded from the request to the relayed request.
+	// The forwarded headers are:
+	// X-Pocket-Supplier, X-Pocket-Service, X-Pocket-Session-Id, X-Pocket-Application
+	// X-Pocket-Session-Start-Height, X-Pocket-Session-End-Height
+	ForwardIdentityHeaders bool
 }
 
```

Comment:
Please update `relayminer_config_ull_example.yaml` with this: https://github.com/pokt-network/poktroll/blob/main/localnet/pocketd/config/relayminer_config_full_example.yaml